packages/eas-cli/src/credentials/android/actions/__tests__/CreateGoogleServiceAccountKey-test.ts

```diff
@@ -0,0 +1,36 @@
+import { vol } from 'memfs';
+
+import { promptAsync } from '../../../../prompts';
+import { createCtxMock } from '../../../__tests__/fixtures-context';
+import { getAppLookupParamsFromContextAsync } from '../BuildCredentialsUtils';
+import { CreateGoogleServiceAccountKey } from '../CreateGoogleServiceAccountKey';
+
+jest.mock('../../../../prompts');
+jest.mock('fs');
+(promptAsync as jest.Mock).mockImplementation(() => ({
+  keyJsonPath: '/google-service-account-key.json',
+}));
+
+describe(CreateGoogleServiceAccountKey, () => {
+  it('creates a Google Service Account Key in Interactive Mode', async () => {
+    vol.fromJSON({
```

Comment:
Add `beforeEach` with `vol.reset()`. It's not necessary now but if we added more tests, this could result in unexpected and hard to debug behavior.